INSTALLATION.md

```diff
@@ -1,218 +1,156 @@
-
-2.  Change directory to cloned folder :-
-
-        cd talawa-api
-
-3.  Talawa-API uses a configuration file named `.env` in the root directory. It is not a part of the repo and you will need to create it. There is a sample configuration file named `.env.sample` in the root directory. Create a new `.env` file by copying the contents of the `.env.sample` file.
+Here are the steps required for setting up the remaining environment variables:-
 
-    cp .env.sample .env
-
-4.  Generate an `ACCESS_TOKEN_SECRET` using the `openessl` command below and copy the result to the `ACCESS_TOKEN_SECRET` section of the `.env` file.
+1. Run the following command and copy/paste the result to the variable named `ACCESS_TOKEN_SECRET` in the `.env` file.
 
         openssl rand -hex 32
 
-5.  Generate an `REFRESH_TOKEN_SECRET` using the `openessl` command below and copy the result to the `REFRESH_TOKEN_SECRET` section of the `.env` file.
+2. Run the following command and copy/paste the result to the variable named `REFRESH_TOKEN_SECRET` in the `.env` file.
 
         openssl rand -hex 32
 
-6.  We have written the docker configuration in such a way that along with talawa-api mongodb is also started inside a docker container. Both of them communicate with each other internally. This is done so you don't have to manually set up a mongodb database and then connect it to the talawa-api docker container yourselves. Within the docker environment this mongodb container is accessible using this connection string :-
 
-        mongodb://talawa_mongodb:27017/<DATABASE_NAME>?retryWrites=true&w=majority
+# Installing required dependencies
 
-    We have mapped this mongodb docker container to port `27017` of your system's localhost. Make sure that port `27017` is free on your system's localhost or you'll get an error. To access this mongodb container on your system use this connection string :-
+Run the following command to install the packages and dependencies required by talawa-api:-
 
-        mongodb://localhost:27017/<DATABASE_NAME>?retryWrites=true&w=majority
-
-7.  When finished, your `.env` file should have the following fields filled in.
-
-    - ACCESS_TOKEN_SECRET
-    - REFRESH_TOKEN_SECRET
-    - MONGO_DB_URL
-
-    Please review the contents of the `.env.sample` file for additional details.
-
-8.  Build the `docker` image that will support talawa-api
-
-        sudo docker-compose build
+        npm run install
 
-9.  To start the talawa-api docker container run this command while being in the talawa-api folder directory :-
 
-        sudo docker-compose up
+# Running talawa-api development server
+Run the following command to run talawa-api development server:-
 
-    **Note** :- There is one caveat here. `Docker` might not automatically start each time you reboot your system. To start docker after a system reboot on linux machines run this command :-
-
-        sudo systemctl start docker
+        npm run dev
```

Comment:
What URL should the Talawa mobile app and Talawa-Admin use when the API is running? This will need to be very clear.

Reply:
Completed.